test/utilities.js

```diff
@@ -37,6 +37,26 @@ let createAndDepositTX = async function(rootchain, address) {
     return [blockNum, confirmHash, confirmSignature, txBytes, txHash, sigs, blockHeader];
 };
 
+let createAndDepositTX2 = async function(rootchain, address) {
```

Comment:
What's the difference between this function and the one above? If there is none, why do we need it?

Reply:
Fixed

Reply:
Rather than having multiple functions, we should refactor createAndDepositTX to take in amount as a parameter.

Reply:
Yeah did exactly that